deploy/monitor/prometheus_rule.yaml

```diff
@@ -22,12 +22,12 @@ spec:
         annotations:
           description: "The unifiedpush-operator has been down for more than 5 minutes. "
           summary: "The unifiedpush-operator is down. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
-          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.2.0/SOP/SOP-operator.adoc" #todo
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc"
```

Comment:
I don't see a tag yet but I guess there will be on at some stage?

Reply:
Yep. It will be done after. Note that when a release be made the tag should be updated and it is described in the README now.

Reply:
👍